index.html

```diff
@@ -0,0 +1,138 @@
+          <img src="https://www.mcdelivery.co.kr/kr//static/1648609546423/assets/82/products/1432.png?" alt="슈슈버거">
+          <div class="burger-detail">
+            <span class="burger-detail-name">슈슈버거</span>
+            <span class="burger-detail-price">₩5,300</span>
+            <span class="burger-detail-ment">얼마나 맛있게요?</span>
+          </div>
+        </article>
+        <article class="burger-card">
+          <img src="https://www.mcdelivery.co.kr/kr//static/1648609546423/assets/82/products/1433.png?" alt="슈비버거">
+          <div class="burger-detail">
+            <span class="burger-detail-name">슈비버거</span>
+            <span class="burger-detail-price">₩6,300</span>
+            <span class="burger-detail-ment">얼마나 맛있게요?</span>
+          </div>
+        </article>
+        <article class="burger-card">
+          <img src="https://www.mcdelivery.co.kr/kr//static/1648609546423/assets/82/products/6310.png?" alt="치즈버거">
+          <div class="burger-detail">
+            <span class="burger-detail-name">치즈버거</span>
+            <span class="burger-detail-price">₩3,100</span>
+            <span class="burger-detail-ment">얼마나 맛있게요?</span>
+          </div>
+        </article>
+        <article class="burger-card">
+          <img src="https://www.mcdelivery.co.kr/kr//static/1648609546423/assets/82/products/1312.png?" alt="더블 치즈버거">
+          <div class="burger-detail">
+            <span class="burger-detail-name">더블 치즈버거</span>
+            <span class="burger-detail-price">₩5,200</span>
+            <span class="burger-detail-ment">얼마나 맛있게요?</span>
+          </div>
+        </article>
+        <article class="burger-card">
+          <img src="https://www.mcdelivery.co.kr/kr//static/1648609546423/assets/82/products/1347.png?" alt="쿼터파운드 치즈">
+          <div class="burger-detail">
+            <span class="burger-detail-name">쿼터파운드 치즈</span>
+            <span class="burger-detail-price">₩6,000</span>
+            <span class="burger-detail-ment">얼마나 맛있게요?</span>
+          </div>
+        </article>
+        <article class="burger-card">
+          <img src="https://www.mcdelivery.co.kr/kr//static/1648609546423/assets/82/products/6169.png?" alt="불고기 버거">
+          <div class="burger-detail">
+            <span class="burger-detail-name">불고기 버거</span>
+            <span class="burger-detail-price">₩3,100</span>
+            <span class="burger-detail-ment">얼마나 맛있게요?</span>
+          </div>
+        </article>
+      </section>
+      <aside class="addToCart">
+        <div class="addToCart-container">
+          <div class="cart">
+            장바구니
+            <div class="cart-menu"></div>
+          </div>
+          <div class="amount">
+            누적금액
+            <span class="amount-price">0원</span>
+          </div>
+          <div class="btn-container">
+            <button class="orderBtn">주문하기</button>
+            <button class="cancelBtn">취소하기</button>
+          </div>
+        </div>
+      </aside>
```

Comment:
```suggestion
      <section class="addToCart">
          <div class="cart">
            <h2>장바구니</h2>
            <div class="cart-menu"></div>
          </div>
          <div class="amount">
            <h3>누적금액</h3>
            <p class="amount-price">0원</p>
          </div>
          <div class="btn-container">
            <button class="orderBtn">주문하기</button>
            <button class="cancelBtn">취소하기</button>
          </div>
      </section>
```
1. 시맨틱 테그 자체를 컨테이너로 쓸 수 있으면 불필요한 div테그를 제거하고 시맨틱테그 자체를 컨테이너로 쓰는 것이 좀 더 좋은 것 같아요! 
2. aside 태그는 본문 흐름에 벗어나는 내용을 의미하는 시멘틱 태그라서, 장바구니는 aside보다 section테그가 조금 더 적절할 것 같아요! 
3. 장바구니, 누적금액, 0원 전부 그냥 텍스트이긴 하지만 최대한 의미를 담을 수 있도록 하는 테그들을 사용하는 게 좋을 것 같아요! 이 section에서 가장 주된 heading이 되는 내용이 '장바구니'니까 h2테그를 쓰고, 누적금액은 그 밑의 h3, 금액은 p테그를 쓰면 span테그보다 좀 더 의미를 담을 수 있을 것 같습니다!

Reply:
중간에 div테그를 제거하면서 스타일을 어떻게 적용해야하나..싶으면 나히찬스.. 써주십쇼